pkg/metrics/eventmetrics/collector.go

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+package eventmetrics
```

Comment:
nit: Add a newline above the package statement, otherwise the SPDX header becomes the package documentation.